src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,40 @@
+package pages;
+
+import enums.BrowserName;
+import org.openqa.selenium.By;
+
+public class LoginPage extends BasePage {
+
+    // Constructor:
+    public LoginPage(BrowserName browserName) {
+        super(browserName);
+    }
+
+    // Locators:
+    private By loginInputLocator = By.xpath("//*[@id=\"loginFrm_loginname\"]");
```

Comment:
Why not simply `By.id`?

Reply:
@dariabajda Answered in the comment above :)